src/function.rs

```diff
@@ -411,3 +415,14 @@ fn without_extension(_context: &FunctionContext, path: &str) -> Result<String, S
 
   Ok(parent.join(file_stem).to_string())
 }
+
+/// Check whether a string processes properly as semver (e.x. "0.1.0")
+/// and matches a given semver requirement (e.x. ">=0.1.0")
+fn semver_matches(_context: &FunctionContext, v: &str, req: &str) -> Result<String, String> {
+  Ok(
+    VersionReq::parse(req.as_ref())
```

Comment:
I like using parse with a turbofish:

```suggestion
    req.parse::<VersionReq>()
```